tailwind.config.js

```diff
@@ -1,6 +1,78 @@
 module.exports = {
   content: ['./src/**/*.{html,js,tsx,ts}'],
-  theme: {},
+  theme: {
+    fontFamily: {
+      sans: ['Nunito', 'sans-serif'],
+      serif: ['Libre Baskerville ', 'serif'],
+    },
+    fontSize: {
+      '4xl': [
+        '3rem',
+        {
+          lineHeight: '1.4',
+        },
+      ],
+      '3xl': [
+        '2.25rem',
+        {
+          lineHeight: '1.4',
+        },
+      ],
+      '2xl': [
+        '1.75rem',
+        {
+          lineHeight: '1.2',
+        },
+      ],
+      l: [
```

Comment:
same here, it's usually called `lg`

Reply:
Taking a look I found there are the same sizes we need, so I deleted and used the tailwind ones